src/ElementQueries.js

```diff
@@ -194,9 +194,12 @@
 
         /**
          * Start the magic. Go through all collected rules (readRules()) and attach the resize-listener.
+         *
+         * @param {HTMLElement} container only elements of the container are considered (document.body if not set)
+         *
          */
-        function findElementQueriesElements() {
-            var query = getQuery();
+        function findElementQueriesElements(container) {
+            var query = (container) ? container.querySelectorAll.bind(container) : getQuery();
```

Comment:
Hey Marc,
you are right, it is only used if document.querySelectorAll is not defined. This only happens, if the browser doesn't support it. According to http://caniuse.com/#feat=queryselector every browser (except IE 8 and below) supports document.querySelectorAll. So jQuery and mootools will never be used. Do you understand what I mean?